pkg/apis/acid.zalan.do/v1/zz_generated.deepcopy.go

```diff
@@ -532,6 +532,11 @@ func (in *PostgresSpec) DeepCopyInto(out *PostgresSpec) {
 		*out = new(bool)
 		**out = **in
 	}
+	if in.EnableReplicaConnectionPooler != nil {
+		in, out := &in.EnableReplicaConnectionPooler, &out.EnableReplicaConnectionPooler
```

Comment:
Why was this code generated ? was there any change to the connection pooler ?

Reply:
Yes, and seem the generated code wasn't updated.

Reply:
not very good, but let's commit it in this PR with a commit message acknowledging this fix